### DISCUSSION TITLE
lfrc file format

### DISCUSSION THREAD

**Discussion by fritzrehde** [gokcehan/lf#775] [Q&A] [answered]
I would like to enable syntax highlighting in vim for the lf config file (lfrc). Vim cannot find the file type automatically, so I have to set it manually. What file type is it?

**Comment by gokcehan** [comment, marked as answer]
@fritzrehde It's a custom format and there are no vim syntax available for it that I'm aware of. Setting `ft=conf` helps to highlight comments and string quotes which is good enough for most purposes. In fact, vim already sets this automatically for files with some comments in the first 5 lines on my system. One additional trick is to set `fdm=marker` and `fmr={{,}}` to fold multiline commands.

**Reply by grimnight** [reply]
There is [camnw/lf-vim].

[camnw/lf-vim]: https://github.com/camnw/lf-vim

**Reply by gokcehan** [reply]
@grimnight I had no idea about it, thanks. It seems great. Maybe we should add it somewhere in the wiki pages.
